hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestIncrementalBackup.java

```diff
@@ -249,4 +377,69 @@ private String takeFullBackup(List<TableName> tables, BackupAdminImpl backupAdmi
     checkSucceeded(backupId);
     return backupId;
   }
+
+  private static void doBulkload(TableName tn, String regionName, byte[]... fams)
```

Comment:
There's already some similar HFile creation code in `TestBulkLoadFiles`. I suggest trying to reuse that code, or merging this code with the existing code. If there's too much difference in functionality, at the very least move these methods into that file.

Reply:
good call, just updated to use the existing functionality, thank you!